addons/web_editor/static/src/js/editor/odoo-editor/src/powerbox/Powerbox.js

```diff
@@ -351,7 +351,13 @@ export class Powerbox {
                     this._context.filteredCommands = this._context.commands.filter(command => {
                         const commandText = (command.category + ' ' + command.name);
                         const commandDescription = command.description.replace(/\s/g, '');
-                        return commandText.match(fuzzyRegex) || commandDescription.match(exactRegex);
+                        const alertSynonyms = ['banner', 'info', 'success', 'warning', 'danger'];
```

Comment:
I don't this is a good idea to implement such things here. It's something command specific filter. what if in future we introduce more keywords ?